assets/src/application/ui/layout/espressoTable/EspressoDragSortingTable/index.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { Table } from 'antd';
+import { DndProvider, DragSource, DropTarget } from 'react-dnd';
```

Comment:
I'm confused... you've installed and imported `react-dnd` here...
but `assets/src/application/ui/layout/espressoTable/table-body.js` is still using `react-beautiful-dnd` ?!?!?

Reply:
As I wrote, I've added `react-dnd`, based on the demo provided by antd: https://github.com/ant-design/ant-design/blob/master/components/table/demo/drag-sorting.md
As it is a straight-forward integration.
***
I've found an issue and a gist based off `react-beautiful-dnd`:
- https://github.com/ant-design/ant-design/issues/19764
- https://gist.github.com/samundrak/98c4868eb6c817ac2b3e52fb57d7b983
Let' loop back on this when draggable table will be tied to actual mutation and see what is most convenient way, I didn't wanted to spend too much time on this now.

Reply:
Also, based on stats I see that `react-beautiful-dnd` is a more popular and newer package, so :heavy_plus_sign: 1 for this lib
![image](https://user-images.githubusercontent.com/1477453/74247873-6befe500-4cef-11ea-8a55-c3de5b50f494.png)